### PR TITLE
Fixes teleportation from inside cryo

### LINKED ISSUE #28651: Cultist summoning doesn't properly remove them form cryo
\>Place cultist in cryo, turn it on
\>Summon them with a rune
\>They remain asleep, build up cryox, and heal as if they're still in the cryotube
\>Doesn't turn off only when fully healed
Occupant var on the cryotube is still set to them, even though they were pulled out, and the sprite isn't updated

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #28651

### CODE REVIEW

**Comment by Bawhoppen** [comment]
speedy cyberboss

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -170,6 +170,9 @@
 		return
 	var/datum/gas_mixture/air1 = AIR1
 	var/turf/T = get_turf(src)
+	if(occupant && occupant.loc != src)
```
Comment (lzimann) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: This feels kinda hacky, isn't the issue cryo not properly removing the occupant?

Comment (AnturK) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Just use Exited.

**Comment by Cyberboss** [comment]
Needs a more general solution

**Pull request closed without merging**